sdks/python/src/opik/integrations/llama_index/event_parsing_utils.py

```diff
@@ -4,9 +4,13 @@
 from llama_index.core import Settings
 from llama_index.core.base.llms.types import ChatResponse
 from llama_index.core.callbacks import schema as llama_index_schema
+from llama_index.core.callbacks import (
```

Comment:
please import modules (I know it was already violated in this file)

Reply:
Was it only for `llama_index` imports or also opik?